suites/reef/rgw/migrate_default_single_to_multisite.yaml

```diff
@@ -0,0 +1,259 @@
+tests:
```

Comment:
Please include the conf file details in test suites

Reply:
done, @anrao19